tests/ui/parser/doc-comment-after-missing-comma-issue-142311.rs

```diff
@@ -0,0 +1,18 @@
+//! Check that the parser does not suggest converting `///` to a regular comment
+//! when it appears after a missing comma in an item list (e.g. `enum` variants).
```

Comment:
```suggestion
//! when it appears after a missing comma in a list (e.g. `enum` variants).
```

Please add a test for arrays too